app/clients/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .metadata import MetadataClient
+from .project import ProjectClient
+
+__all__ = [MetadataClient, ProjectClient]
```

Comment:
Elements should be strings.
```suggestion
__all__ = [
  'MetadataClient', 
  'ProjectClient',
]
```